src/Reshuffle.ts

```diff
@@ -18,12 +19,21 @@ export default class Reshuffle {
     handlers: { [id: string]: Handler[] }
     common: { webserver?: Express; persistentStore?: any }
   }
+  logger: Logger
 
-  constructor() {
+  constructor(useLogger = true) {
     this.availableConnectors = availableConnectors
     this.port = parseInt(<string>process.env.PORT, 10) || 8000
     this.httpDelegates = {}
     this.registry = { connectors: {}, handlers: {}, common: {} }
+    this.logger = new Logger()
+    if (useLogger) {
```

Comment:
this should be moved to enterprise.

Reply:
Amir and I decided on keeping this at the end as Watson is a standard log package and allow the user to filter what he wants to be displayed in the console/log files